app/content/carto-user-manual/settings/managing-user-groups.md

```diff
@@ -0,0 +1,32 @@
+## Managing user groups
+
+When a Single Sign-On (SSO) is configured to access CARTO, we can also synchronize the groups coming from your Identity Provider (IdP) after each user's login.
+
+{{% bannerNote title="Note" type="note"%}}
+Single Sign-On and Groups integrations are only available for Enterprise Medium plans and above. Please get in touch at hello@carto.com if you're interested in this feature.
+{{%/ bannerNote %}}
+
+![Settings groups list](/img/cloud-native-workspace/settings/settings_groups_list.png)
```

Comment:
I can give you an updated img tomorrow morning, Sergio is working on it. For example, company is wrongly written and "Read more in the documentation" should be in blue.

Reply:
Sure, thank you!